egs/wsj/s5/utils/scoring/append_prf_to_ctm.py

```diff
@@ -0,0 +1,74 @@
+#!/bin/env python
+
+# Copyright 2015  Brno University of Technology (author: Karel Vesely)
+# Apache 2.0
+
+import sys
+import numpy as np
+
```

Comment:
Karel, you won't like to hear this, but the numpy dependency is problematic.
numpy is not very easy to install, and tends to cause version-mismatch problems.  And it seems unnecessary here as you're only using it for parsing, not for any mathematical stuff.
I'd be OK with keeping the numpy dependency only if we downgrade this to a setup-specific script in 'local'/.